### PR TITLE
Minor Update to README.srt

### PR DESCRIPTION
**Pull request by headchefxander**
- fixed few spelling errors
- fixed few punctuation errors
- fixed missing or wrong preposition
- improved conciseness in few sentences

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -55,14 +55,14 @@ internet.
...
 ecosystem.
 
 In Matrix, every user runs one or more Matrix clients, which connect through to
-a Matrix homeserver. The homeserver stores all their personal chat history and
-user account information - much as a mail client connects through to an
-IMAP/SMTP server. Just like email, you can either run your own Matrix
-homeserver and control and own your own communications and history or use one
+a Matrix homeserver. The homeserver stores all of the personal chat history and
+user account information of users - much as a mail client connects through to an
+IMAP/SMTP server. For example, email, you can either run your Matrix
```
Comment (aaronraimist) [inline README.rst]: This line doesn't make sense now. I think the "Just like" worked fine.

File: README.rst [hunk 2]
```diff
@@ -55,14 +55,14 @@ internet.
...
+homeserver to control your communications and history or use the one
```
Comment (aaronraimist) [inline README.rst]:
```suggestion
homeserver to control your communications and history or use the
```

File: README.rst [hunk 3]
```diff
@@ -95,9 +95,9 @@ Installing from source
 (Prebuilt packages are available for some platforms - see `Platform-Specific
 Instructions`_.)
 
-Synapse is written in Python but some of the libraries it uses are written in
-C. So before we can install Synapse itself we need a working C compiler and the
-header files for Python C extensions.
+Synapse is written in Python, but some of the libraries it uses are written in C. 
+Before we can install the Synapse, we need a working C compiler and the header files
```
Comment (aaronraimist) [inline README.rst]:
```suggestion
Before we can install Synapse, we need a working C compiler and the header files 
```

Not the, just Synapse

File: README.rst [hunk 4]
```diff
@@ -152,11 +152,11 @@ To install the Synapse homeserver run::
...
 
-This installs Synapse, along with the libraries it uses, into a virtual
-environment under ``~/.synapse``.  Feel free to pick a different directory
-if you prefer.
+The above installation of Synapse along with the libraries it uses are stored 
+in a virtual environment under the directory ~/.synapse. 
+Feel free to pick a different directory if you prefer.
 
-This Synapse installation can then be later upgraded by using pip again with the
+The installed Synapse can then be upgraded by using pip with the
```
Comment (aaronraimist) [inline README.rst]: For lines 155-159 I actually prefer the old version

**Comment by aaronraimist** [comment]
You also need to add a newsfragment file so the tests complete properly. See https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst#changelog

**Comment by headchefxander** [comment]
Roger that sir, I will change my edits as you said. 
Btw, for the tests, I am a university student currently taking a writing course, the edit to README.srt is part of the course work. The instructor asked us to do a real-world edit to README file only. Is it fine that I only change the edits I made?

**Comment by richvdh** [comment]
> it fine that I only change the edits I made?

I don't really understand this question. If you'd like to contribute changes to this project, you'll need to follow the contribution rules which, as Aaron explained,  include writing a newsfragment file, basing the pull-request on the right branch, and signing off your contribution.

**Comment by richvdh** [comment]
feel free to reopen if you come back to this

**Pull request closed without merging**